src/assets/styles/components/sidebar.scss

```diff
@@ -0,0 +1,170 @@
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--surface);
+  font-size: 14px;
+  color: var(--text);
+  width: 16px;
+  overflow: hidden;
+  height: 100vh;
+  transition: width 0.3s ease;
+}
+
+.sidebar.expanded {
+  width: 260px;
+}
+
+.sidebar .expand-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: transparent;
+  border: none;
+  padding:0;
+  color: #fff;
+  z-index: 100;
+  cursor: pointer;
+  transition: left 0.3s ease, background-color 0.3s ease, border-radius 0.3s ease, padding 0.3s ease;
+}
+
+.sidebar .expand-button.collapsed {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  border-radius: 3px;
+  background-color: var(--surface);
+  border: 1px solid var(--border);
+  position: absolute;
+  top: 10px;
+  left: 0;
+  border-radius: 50%;
+}
+
+.sidebar .expand-button.collapsed:hover {
+  background-color: var(--dynamic-button-hovered);
+}
+
+
+.sidebar .expand-button.collapsed img {
+  rotate: 180deg;
+}
+
+
+
+.avatar {
+  width: 40px;
+  height: 40px;
+  background-color: #555;
+  border-radius: 5px;
+  margin:5px;
+}
+
+.user-info {
+  display:flex;
+  flex-direction: column;
+  font-size: 14px
+}
+
+.username {
+  font-weight: 600;
+}
+
+.workspace-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 600;
+  padding:8px;
+}
+
+.workspace-container:hover {
+  
+  .icon-details {
+    display: block;
+    padding: 4px;
+  }
+
+}
+
+.list-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.list-item {
+  display: flex;
+  align-items: center;
+  font-style: italic;
+  font-size: 14px;
+  padding: 0 12px;
+  height: 32px;
+  gap: 10px;
+  width: 100%;
+}
+
+.list-item:hover {
+  background-color: var( --ds-background-neutral-hovered);;
+  border-radius: 2px;
+  transition: 0.3s;
+
+     .icon-details {
+        display: block;
+        cursor: pointer;
+  }
+
+     .icon-star {
+        display: block;
+        cursor: pointer;
+     }
+
+    .icon-details:hover {
+    background-color: var( --ds-background-neutral-hovered);
+    border-radius: 2px;
+    cursor: pointer;
+    transition: 0.3s;
+  }
+
+    .icon-star:hover {
+      cursor: pointer;
+      transition: 0.3s;
```

Comment:
you dont need to add cursor pointer every time, is ok just on the class hover dont need.
also transition move to the class not in the hover

Reply:
recommend to add all transition props.